.gitignore

```diff
@@ -20,4 +20,6 @@ tmp
 *.o
 *.a
 mkmf.log
-gemfiles/*.lock
\ No newline at end of file
+gemfiles/*.lock
+*.ruby-*
```

Comment:
Also, today I learned there is a ~/.gitignore.  I should have never pushed this file... :/ sorry.